### ISSUE TITLE
alasql.js:19270 Uncaught TypeError: SQL.Database is not a constructor

### ISSUE THREAD

**Issue by carper7** [AlaSQL/alasql#1284]
I get this error when trying to attach a database from the disk. This is the code: 

```
alasql(['ATTACH SQLITE DATABASE inscriptions("mydb.sqlite3");\
            Use inscriptions; \
            select * from signatures']).then(function(res) {
                console.log("Signatures: ", res.pop());
});	
```

I am using only javascript and html.

**Comment by mathiasrw** [comment]
Hmmm. 

Does it help if you do this?

```
alasql(['ATTACH SQLITE DATABASE inscriptions("mydb.sqlite3")',
        'Use inscriptions',
        'select * from signatures']).then(function(res) {
                console.log("Signatures: ", res.pop());
});	
```

**Comment by carper7** [comment]
Hi. Thank you for getting back to me on this. Unfortunatelly it does not work; this the error I am getting now :

alasql.js:15448 Uncaught (in promise) Error: Database 'inscriptions' does not exist and again after that  alasql.js:19270 Uncaught TypeError: SQL.Database is not a constructor

**Comment by mathiasrw** [comment]
Ahh yes. We have a bug on the USE part where it cant be in the same call as the others. 

what about this one?

```
await alasql(['ATTACH SQLITE DATABASE inscriptions("mydb.sqlite3")']);

await alasql(["Use inscriptions"]);

let res = await alasql(["select * from signatures"]);

console.log("Signatures: ", res.pop());
```

**Comment by carper7** [comment]
Hi .

Sorry but that did not make a difference. I got same error as before:

alasql.js:19270 Uncaught TypeError: SQL.Database is not a constructor




________________________________

**Comment by 34ERAT** [comment]
Hey, is this issue still open? I am new to open source and will like to contribute to this

**Comment by mathiasrw** [comment]
@34ERAT Yep. Its stil open. Ill assign it to you. Let me know if you need guidance.

**Comment by mathiasrw** [comment]
@34ERAT Are you still happy to be assigned?

**Comment by 34ERAT** [comment]
> @34ERAT Are you still happy to be assigned?
Thanks  , i am working  on the issue  this  coming weekend  , i am really  excited

**Comment by mathiasrw** [comment]
@34ERAT Anything I can help with?

**Comment by 34ERAT** [comment]
@mathiasrw I am having issues. Replicating the error, I can't seem to be able to reproduce it

**Comment by mathiasrw** [comment]
Fix released as part of v4.19.0
